src/coffee/stockimport.coffee

```diff
@@ -90,7 +89,7 @@ class StockImport
         ElasticIo.returnFailure err, next
       .done()
     else
-      ElasticIo.returnFailure "#{LOG_PREFIX}No data found in elastic.io msg.", next
+      ElasticIo.returnFailure "#{CONS.LOG_PREFIX}No data found in elastic.io msg.", next
```

Comment:
FYI: to be honest, everything related to `elastic.io` can be removed. Check with @hajoeichler first please